app/models/cell_metadatum.rb

```diff
@@ -71,12 +71,18 @@ def annotation_select_option
 
   def can_visualize?
```

Comment:
looking through the codebase, this method looks like it's is always accessed in a keep_if? loop.  So if performance matters, it might be worth making a study-level 'visualizable_metadata' method that grabs all the visualizable fields in one go without requiring a DB query per column.  But that's a non-trivial query to write, so it may not be worth it.

Reply:
That's a really good point - I think this method still has a place (specifically the "study info" page), but in the context of getting all visualizable annotations for a study, we want to optimize that.  I'll work on that refactoring.

Reply:
40a51e6